metacat-common/src/main/java/com/netflix/metacat/common/type/Base.java

```diff
@@ -73,24 +73,23 @@
      *  nnnnnn: optional nanotime
      *  example INTERVAL '1 2:3:4.000005' DAY.
      * */
-    INTERVAL_DAY_TO_SECOND("interval day to second", false, false),
+    INTERVAL_DAY_TO_SECOND("interval day to second", false),
 
     /** unknown type. */
-    UNKNOWN("unknown", false, false),
+    UNKNOWN("unknown", false),
     /** array type. */
-    ARRAY("array", false, true),
+    ARRAY("array", true),
     /** row type. */
-    ROW("row", false, true),
+    ROW("row", true),
     /** map type. */
-    MAP("map", false, true);
+    MAP("map", true);
 
     private final String type;
     private boolean isParametricType;
-    private boolean isComplexType;
-    Base(final String type, final boolean isParametricType, final boolean isComplexType) {
+
+    Base(final String type, final boolean isParametricType) {
```

Comment:
We should rename this class to better represent what it is. `Base` is far too generic of a name. Perhaps `TypeEnum` or similar.